src/App.js

```diff
@@ -1,45 +1,28 @@
 
-import "./App.css";
-import logo from "./logo.png";
+import Navbar from "./Components/Navbar/Navbar-comp";
+import Wishlist from "./Components/Wishlist/Wishlist";
+import Product from "./Product/Product";
+import Authentication from "./Components/Authentication/Login"
+import Cart from "./Components/Cart/Cart";
```

Comment:
As all of these imports are coming from `/components` folder so you could create one `index.js` file in the components folder and export all of these components from that index.js file. With this you get the advantage of importing all the components together.

e.g.
index.js
```jsx
export {Navbar} from "./Navbar/Navbar-comp";
export {Wishlist} from "./Wishlist/Wishlist";
export {Authentication} from "./Authentication/Login";
export {Cart} from "./Cart/Cart";
```

and importing in `App.js` would look like
```jsx
import {Navbar, Wishlist, Authentication, Cart} from "./Components";
```